packages/kotti-ui/source/next/kotti-field/components/FieldHelpText.vue

```diff
@@ -0,0 +1,67 @@
+<script lang="ts">
+import { Yoco } from '@3yourmind/yoco'
+import { defineComponent, ref } from '@vue/composition-api'
+import { roundArrow } from 'tippy.js'
+import { VNode } from 'vue'
+import { useTippy } from 'vue-tippy/composition'
+
+const ARROW_HEIGHT = 7
+
+export default defineComponent<{
+	helpText: string | null
+	helpTextSlot: VNode[]
+}>({
+	name: 'FieldHelpText',
+	props: {
+		helpText: { default: null, type: String },
+		helpTextSlot: { default: () => [], type: Array },
+	},
+	setup() {
+		const helpTextContentRef = ref<Element | null>(null)
+		const helpTextTriggerRef = ref<Element | null>(null)
+
+		useTippy(helpTextTriggerRef, {
+			appendTo: () => document.body,
```

Comment:
for future: not the default if `interactive` is set to true, read the docs THOROUGHLY